doc/extdev/tutorials/todo_ext.rst

```diff
@@ -1,7 +1,7 @@
-.. _exttut:
```

Comment:
I don't think all of this should be moved. Instead, we should only move everything from the "Extension Design" line below. The rest of this should go into `development/index.rst`.

In addition, instead of calling this `todo_ext`, could we call it `todo` and put it in the `development/tutorials/` directory noted earlier?